Heap (Actions) destination docs (#3187)
segmentio/segment-docs: src/connections/destinations/catalog/actions-heap/index.md

---
title: Heap (Actions) Destination
hide-boilerplate: true
hide-dossier: true
hidden: true
private: true
id: 62b256147cbb49302d1486d0
versions:
  - name: Heap (Classic)
    link: /docs/connections/destinations/heap
---

{% include content/plan-grid.md name="actions" %}

> [Heap](https://heapanalytics.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) automatically captures every user interaction with no extra code. This includes clicks, taps, gestures, form submissions, page views, and more.

Reviewer: ```suggestion
[Heap](https://heapanalytics.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} automatically captures every user interaction with no extra code. This includes clicks, taps, gestures, form submissions, page views, and more.
```

> success ""

> > **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Heap destination. There's also a page about the [non-Actions Heap destination](/docs/connections/destinations/catalog/heap/). Both of these destinations receives data from Segment.

Reviewer: ```suggestion
> **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Heap destination. There's also a page about the [non-Actions Heap destination](/docs/connections/destinations/catalog/heap/). Both of these destinations receive data from Segment.
```

{% include content/ajs-upgrade.md %}

## Benefits of Heap (Actions) vs Heap Classic

Heap (Actions) provides the following benefits over the classic Heap destination:

- **More control with clearer mapping**. Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to the destination.

> - **Increased transparency**. All Segment ingested events will display the Segment icon in Heap to easily identify the source of the event, and foster greater data trust

Reviewer: ```suggestion
- **Increased transparency**. All Segment ingested events display the Segment icon in Heap, so you can identify the source of the event, and foster greater data trust.
```

> - **Improved setup**. Easier default settings make it easier to get set up faster

Reviewer: ```suggestion
- **Improved setup**. Clear default settings enable you to set up faster.
```

## Getting started

> 1. From the Segment web app, click **Catalog**, then click **Destinations**.

Reviewer: ```suggestion
1. From the Segment web app, navigate to **Connections** and click **Add Destination**.
```

> 2. Find the Destinations Actions item in the left navigation, and click it.

Reviewer: ```suggestion
2. Search for **Heap (Actions)** and click it.
```

Reviewer: I'm guessing it will be called "Heap (Actions)"?

3. Click **Configure Heap**.

> 4. Select an existing Source to connect to Heap (Actions).

Reviewer: ```suggestion
4. Select the Source to connect to Heap (Actions).
```
